### PR TITLE
Add Community Guidelines

### PR DESCRIPTION
**Pull request by karst**
Depends on #21087

### CODE REVIEW

**Review by IntelOrca — APPROVED** [review]

**Comment by karst** [comment]
😂 how does it fail on mac when I dont even touch the game's code

**Comment by ZehMatt** [comment]
You can probably skip the CI, if I recall correctly you can add [ci skip] in the commit message, it shouldn't have to build for this.

**Review by janisozaur — APPROVED** [review]

File: CommunityGuidelines.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - Do not post links to sites/content others would find unacceptable.
+- Advertising or promotion of piracy is forbidden.
+- Staff members (`@Developer` on Discord) have the final say about what is and is not acceptable within the community. Refusals to listen to staff may result in being kicked or banned.
+- HAVE FUN and enjoy all that the community has to offer.
+
+To read our Code of Conduct please refer to [Code of Conduct](CODE_OF_CONDUCT.md).
+ 
+## Team rules
+
+On top of the community rules the development and moderation team has to keep themselves to a specific set of rules.
```
Comment (tupaschoal) [inline CommunityGuidelines.md:20]:
```suggestion
On top of the community rules, the development and moderation team has to keep themselves to a specific set of rules.
```

**Review by tupaschoal — APPROVED** [review]

File: CommunityGuidelines.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+| ---- | --------- |
+| Ted (IntelOrca) | ~/i | 
+| Duncan (duncanpumpkin) | ~/d |
+| Michał (janisozaur) | ~/j |
+| Hielke (Broxzier) | ~/b |
+| Tulio (tupaschoal) | ~/t | 
+| Olivier (Oli414) | ~/o |
+| Matthias (ZehMatt) | ~/z |
+| Karst (AuraSpecs)  | ~/a |
+| Michael (mrmbernardi) | ~/m |
```
Comment (Broxzier) [inline CommunityGuidelines.md:60]: Time to add @Gymnasiast now too!
Also interesting how there are still no conflicts on the first letters

```suggestion
| Michael (mrmbernardi) | ~/m |
| Michael (Gymnasiast) | ~/g |
```

**Review by ZehMatt — APPROVED** [review]

File: CommunityGuidelines.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+### Behavioural rules
+
+- Keep cool to any community member, even if they can be very frustrating to deal with: Do not retaliate, say nothing if you cannot keep it polite.
+- Stand up for one another and take responsibility for each others mistakes.
+- Any team member can request a time-out in team chat, which will suspend any talking in team chat for 30 minutes.
+  - Respect the time out, do not contact each other in DM during this period.
+- We work with a 3 strike system, if you hit strike 3 you will lose your role.
+
+### Rules regarding PRs
+- Any contributor PR requires at least two code reviews, either by developers or trusted community members (at least one developer).
```
Comment (duncanspumpkin) [inline CommunityGuidelines.md:31]: missing a new line here.

File: CommunityGuidelines.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- Any message* for support or public relations requires a signature**.
+- When sending emails, finish the letter with: Your regards, your name and the OpenRCT2 Team.
+- Please keep a professional tone when communicating for support and public relations purposes.
+
+*Discord messages are already signed.  
+**For the identity associated with each signature refer to the paragraph "Team signatures"
+
+
+#### Team Signatures
+When finding a message on our social media the following signatures refer to the following members:
```
Comment (duncanspumpkin) [inline CommunityGuidelines.md:48]: and here

**Comment by tupaschoal** [comment]
@karst seems like the only thing missing in here is to tend to Duncan's comments, since the dependency has already been merged. Can you please check?

File: CommunityGuidelines.md (hunk 3 above)
Comment (tupaschoal) [reply CommunityGuidelines.md:31]:
```suggestion

- Any contributor PR requires at least two code reviews, either by developers or trusted community members (at least one developer).
```

File: CommunityGuidelines.md (hunk 4 above)
Comment (tupaschoal) [reply CommunityGuidelines.md:48]:
```suggestion

When finding a message on our social media the following signatures refer to the following members:
```

**Pull request merged**